src/nginx/azext_nginx/aaz/latest/nginx/deployment/certificate/_create.py

```diff
@@ -0,0 +1,313 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "nginx deployment certificate create",
+)
+class Create(AAZCommand):
+    """Create a certificate for an Nginx deployment
+
+    parameters:
+          - name: --certificate-path
+            This path must match one or more ssl_certificate_key directive file argument in your Nginx configuration. This path must be unique between certificates within the same deployment
+          - name: --key-path
+            This path must match one or more ssl_certificate directive file argument in your Nginx configuration. This path must be unique between certificates within the same deployment
+          - name: --key-vault-secret-id
+            The secret ID for your certificate from Azure Key Vault
```

Comment:
Parameter help message should be in the argument short-summery or long-summery in aaz-dev-tools.

Reply:
Done